docs/01_quick_start/01_hydration.md

```diff
@@ -3,12 +3,12 @@ title: What is Hydration?
 sidebar_position: 0
 ---
 
-Hydration is a next-gen DeFi protocol built on Polkadot. Its mission is to **make DeFi efficient, simple, and unstoppable**. To achieve this, Hydration **unites swaps, lending and a stablecoin currency** under the roof of a single, scalable appchain.
+Hydration is a next-gen DeFi protocol built on Polkadot. Its mission is to make DeFi efficient, simple, and unstoppable. To achieve this, Hydration unites swaps, lending and a stablecoin currency under the roof of a single, scalable appchain.
 
-**The Omnipool lies at the heart of trading on Hydration**. It is an innovative Automated Market Maker (AMM) that combines all assets in a single trading pool, unlocking unparalleled efficiencies. By eliminating liquidity fragmentation, the Omnipool enables lower slippage and fewer hops, resulting in capital efficiency gains for traders.
+At the heart of Hydration lies the innovative **[Omnipool](/products/trading/pools/omnipool)**, an Automated Market Maker (AMM) that fundamentally changes how liquidity works in DeFi. Rather than fragmenting liquidity across countless trading pairs, the Omnipool combines all assets into a single trading pool, enabling lower slippage, fewer hops between trades, and unprecedented capital efficiency gains. This breakthrough design also introduces **[1-sided liquidity provisioning](docs/02_products/01_trading/03_liquidity/01_single_sided_lp.md),** making it easier than ever for users to participate in market making.
```

Comment:
It is not just Omnipool, we need to convey the message here "we have the right AMM for any job" - including Stableswap (with 1:1 peg), Stableswap with drifting peg, Isolated Pools and Liquidity Bootsrapping Pools.

And then say a few words about the unique features of Omnipool